packages/scss/src/components/inlineMessage/mods.scss

```diff
@@ -3,8 +3,9 @@
 @mixin S {
 	--components-inlineMessage-fontSize: var(--sizes-XS-fontSize);
 	--components-inlineMessage-lineHeight: var(--sizes-XS-lineHeight);
+	--components-inlineMessage-icon-offset: 1px;
```

Comment:
Not sure why it replaced the default 2px as it appears not aligned now.
Maybe related to browser and/or screen?